JS: Add django template urls as "save urls" (#14943)
github/codeql: javascript/ql/src/change-notes/2023-11-28-django-urls.md

---
category: minorAnalysis
---

> * Added django URLs to detected "safe" URL patterns beside flask and nunjucks

Reviewer: ```suggestion
* Added django URLs to detected "safe" URL patterns in `js/unsafe-external-link`. 
```
